lib/cfd_trading/cfd_overview.dart

```diff
@@ -22,8 +23,10 @@ class CfdOverview extends StatefulWidget {
 class _CfdOverviewState extends State<CfdOverview> {
   @override
   Widget build(BuildContext context) {
+    final cfdOffersChangeNotifier = context.watch<CfdOfferChangeNotifier>();
```

Comment:
🛠️ This is a duplicate of line `27`! We should just remove the one named `service`

Reply:
Don't think so. One is for CFDs and the other is for the offer.

Reply:
Fair enough, I mixed that up!